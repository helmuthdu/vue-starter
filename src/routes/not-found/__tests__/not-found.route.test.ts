import { mount, VueWrapper } from '@vue/test-utils';
import NotFoundRoute from '../not-found.route.vue';

describe('Route -> NotFound', () => {
  let wrapper: VueWrapper<any>;

  beforeEach(() => {
    wrapper = mount(NotFoundRoute);
  });

  it('should renders with props', () => {
    expect(wrapper.exists()).toBe(true);
  });
});
