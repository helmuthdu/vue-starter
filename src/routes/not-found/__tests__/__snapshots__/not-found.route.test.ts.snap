// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Route -> NotFound should renders with props 1`] = `
VueWrapper {
  "__app": Object {
    "_component": Object {
      "__emits": Object {},
      "__props": Array [
        Object {},
        Array [],
      ],
      "name": "VTU_ROOT",
      "render": [Function],
    },
    "_container": <div
      data-v-app=""
      id="app"
    >
      <div>
        <h1>
          You are here!
        </h1>
        <h2>
          But nothing found for you #404
        </h2>
      </div>
    </div>,
    "_context": Object {
      "app": [Circular],
      "components": Object {},
      "config": Object {
        "errorHandler": undefined,
        "globalProperties": Object {},
        "isCustomElement": [Function],
        "isNativeTag": [Function],
        "optionMergeStrategies": Object {},
        "performance": false,
        "warnHandler": undefined,
      },
      "directives": Object {},
      "mixins": Array [
        Object {
          "__emits": null,
          "__props": Array [],
          "beforeCreate": [Function],
        },
        Object {
          "__emits": null,
          "__props": Array [],
          "beforeCreate": [Function],
        },
      ],
      "provides": Object {},
      "reload": [Function],
    },
    "_props": null,
    "_uid": 0,
    "component": [Function],
    "config": Object {
      "errorHandler": undefined,
      "globalProperties": Object {},
      "isCustomElement": [Function],
      "isNativeTag": [Function],
      "optionMergeStrategies": Object {},
      "performance": false,
      "warnHandler": undefined,
    },
    "directive": [Function],
    "mixin": [Function],
    "mount": [Function],
    "provide": [Function],
    "unmount": [Function],
    "use": [Function],
    "version": "3.0.7",
  },
  "__isFunctionalComponent": false,
  "__setProps": [Function],
  "componentVM": Object {
    "__emitted": Object {},
  },
  "rootVM": Object {
    "__emitted": Object {},
  },
}
`;
